src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -321,6 +324,7 @@ const wchar_t* mainWindowTitle = L"Endless Installer";
 
 #define ALL_FILES					L"*.*"
 
+#define CODING_INSTALLER_NAME    L"coding-installer.exe"
```

Comment:
As for the uninstaller, I remember talking about that being fine for a first pass, and that if needed we would address that with a separate PR.